charts/firecloudorch/templates/deployment.yaml

```diff
@@ -157,6 +157,9 @@ spec:
         livenessProbe:
           {{- toYaml .Values.probes.liveness.spec | nindent 10 }}
         {{- end }}
+        startProbe:
+          {{- toYaml .Values.probes.liveness.spec | nindent 10 }}
```

Comment:
values here should be `.Values.probes.startup.spec`

Reply:
c1eed21